bucket/algernon.json

```diff
@@ -1,25 +1,25 @@
 {
-    "version": "1.12.14",
+    "version": "1.13.0",
     "description": "Small self-contained pure-Go web server with Lua, Markdown, HTTP/2, QUIC, Redis and PostgreSQL support",
     "homepage": "https://algernon.roboticoverlords.org",
     "license": "MIT",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/xyproto/algernon/releases/download/1.12.14/algernon-1.12.14-windows.zip",
-            "hash": "c0a1e05ed3178585221fe79d11307eee96f59ce0b1caad26dc1d2ab6ddd29379"
+            "url": "https://github.com/xyproto/algernon/releases/download/v1.13.0/algernon-1.13.0-windows_x86_64_static.zip",
+            "hash": "4919f60d879b2707d6773954c53eeb8ad1ade49cd67a1eedf63d2692a8bcdcfd"
         }
     },
-    "extract_dir": "algernon-1.12.14",
+    "extract_dir": "algernon-1.13.0-windows_x86_64_static",
     "bin": "algernon.exe",
     "checkver": {
         "github": "https://github.com/xyproto/algernon"
     },
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://github.com/xyproto/algernon/releases/download/$version/algernon-$version-windows.zip"
+                "url": "https://github.com/xyproto/algernon/releases/download/v$version/algernon-$version-windows_x86_64_static.zip"
             }
         },
-        "extract_dir": "algernon-$version"
+        "extract_dir": "algernon-$version-windows_x86_64_static"
```

Comment:
This line should go in the arch 64bit block now

Reply:
Done.